nekoyume/Assets/_Scripts/UI/Module/Button/BonusBuffButton.cs

```diff
@@ -62,23 +68,24 @@ public void SetData(CrystalRandomSkillState skillState, int currentStageId)
                 return;
             }
 
-            if (skillState is null)
+            var starCount = skillState != null ? skillState.StarCount : 0;
+            _hasEnoughStars = starCount >= row.MaxStar;
+            if (_hasEnoughStars)
             {
-                _hasEnoughStars = false;
-                starCountText.text = $"0/{row.MaxStar}";
-                gameObject.SetActive(true);
-                return;
+                _disposableForOnDisabled?.Dispose();
+                _disposableForOnDisabled = Widget.Find<BuffBonusResultPopup>().OnBuffSelectedSubject
+                    .Subscribe(_ => SetIcon(skillState))
+                    .AddTo(gameObject);
```

Comment:
You don't need add `IDisposable` to `gameObject` when you handle it other way with `_disposableForOnDisabled`.

```suggestion
                _disposableForOnDisabled?.Dispose();
                _disposableForOnDisabled = Widget.Find<BuffBonusResultPopup>().OnBuffSelectedSubject
                    .Subscribe(_ => SetIcon(skillState));
```